lldb/test/API/tools/lldb-dap/variables/TestDAP_variables.py

```diff
@@ -153,10 +153,18 @@ def do_test_scopes_variables_setVariable_evaluate(
         buffer_children = make_buffer_verify_dict(0, 32)
         verify_locals = {
             "argc": {
-                "equals": {"type": "int", "value": "1"},
-                "declaration": {
-                    "equals": {"line": 12, "column": 14},
-                    "contains": {"path": ["lldb-dap", "variables", "main.cpp"]},
+                "equals": {
+                    "type": "int",
+                    "value": "1",
+                },
+                "$__lldb_extensions": {
+                    "equals": {
+                        "lldbValue": "1",
```

Comment:
`rawValue` or just `value` now that this is in the `$__lldb_extensions` key? I assume this is the raw `SBValue::GetValue()` only here? Does this not show up if there is no value (structs and class instances have no values). We could mark it as optional if it isn't already

Reply:
Yes, it's the same as SBValue::GetValue()